python/az/aro/azext_aro/_params.py

```diff
@@ -18,6 +19,9 @@
 from azext_aro._validators import validate_version_format
 from azext_aro._validators import validate_outbound_type
 from azext_aro._validators import validate_load_balancer_managed_outbound_ip_count
+from azext_aro._validators import validate_enable_managed_identity
```

Comment:
nit: we can use `import (...)` just as `python/az/aro/azext_aro/tests/latest/unit/test_validators.py`

Reply:
I made this change globally across our imports for consistency's sake.